pkg/kubelet/config/file_linux.go

```diff
@@ -40,7 +41,15 @@ const (
 	podDelete
 )
 
-func (s *sourceFile) watch() error {
+func (s *sourceFileListerWatcher) watch() {
+	go wait.Forever(func() {
+		if err := s.doWatch(); err != nil {
+			glog.Errorf("unable to read config path %q: %v", s.path, err)
+		}
+	}, s.period)
```

Comment:
Why use `s.period` here? `doWatch` contains a watch loop, so this code here should only loop on error. I'd expect the error to be handled, and the watch to be called immediately.